modules/docs/src/03-protocols/04-deriving-cli.md

```diff
@@ -39,7 +40,8 @@ structure Greeting {
 }
 ```
 
-Lets implement the HelloWorld service
+Lets implement the HelloWorld service , we will use cats.effect.IO for our effect type.
+
 ```scala
```

Comment:
I'd like for the examples in this documentation to be using `mdoc` to validate that the code is indeed valid.

Reply:
I will look into how to do that